src/pages/class/[cid]/question/create.tsx

```diff
@@ -63,7 +63,7 @@ export default function Page() {
           similarOptions: [],
         })
 
-        push('/class/' + cid + '/question/' + res.data + '/create-option')
+        push('/class/' + cid)
```

Comment:
perhaps we should route to a specific topic
```suggestion
        push(`/class/${cid}?topic=${topic}`)
```

Reply:
> perhaps we should route to a specific topic

Oh good idea! I have updated it here https://github.com/kixlab/kuiz/pull/60/commits/5787b8d1ce71efc5392211c5eb9bb68ccff0252a :)